resource-rating-plugin-functionality/resource-rating-plugin-meta.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ *
+ */
+function resource_rating_uninstall () {
+	$post_ids = get_posts( array(
+		'numberposts'   => -1,
+		'fields'        => 'ids',
+		'post_type'     => array('post', 'page'),
```

Comment:
You would probably need to a way to specify multiple post types, incase the user is using a different post type, apart from `page` and `post`.